config/users.yml

```diff
@@ -60,3 +50,7 @@
 - email: "paul.dougan@digital.cabinet-office.gov.uk"
   origin: "google"
   deploy_envs: ['prod', 'prod-lon', 'stg-lon']
+
+- email: "emma.pearce@digital.cabinet-office.gov.uk"
+  origin: "google"
+  deploy_envs: ['prod', 'prod-lon', 'stg-lon']
```

Comment:
I know the default is `cf_admin: false`, but I'm debating weather it may be beneficial to make it clear.